docs/components/tooltip.md

```diff
@@ -36,8 +36,8 @@ KTooltip won't be triggered if the element you pass through `default` slot has `
 
 <KCard>
   <template #body>
-    <div class="d-flex">
-      <KTooltip label="I won't pop up" class="mr-2">
+    <div class="k-tooltip">
```

Comment:
Could you not use `k-tooltip` class in the docs? Make it something more generic, like maybe `my-tooltip`  so it doesn't overlap with component styles. Same for `k-tooltip-label`

Reply:
updated all occurrences to generic lass names